### DISCUSSION TITLE
May I ask if pybop supports the P2D model?

### DISCUSSION THREAD

**Discussion by YoungAgile** [pybop-team/PyBOP#278] [Q&A] [answered]
Currently, it seems that I have only seen the SPM model in the example.

**Comment by BradyPlanden** [comment, marked as answer]
Yes, as of today PyBOP supports the P2D as well as the MPM and MSMR models. See: https://github.com/pybop-team/PyBOP/pull/250 for more details. To access these models, you will need to pip install pybop from the develop branch until the next release. This is accomplished by:
```bash
pip install git+https://github.com/pybop-team/PyBOP.git@develop
```

**Reply by YoungAgile** [reply]
Thank you for your reply. I will try it out.

**Reply by BradyPlanden** [reply]
Excellent, do let us know if you run into any issues or brittle areas!
